package.json

```diff
@@ -8,7 +8,9 @@
     "coverage": "nyc npm run unit",
     "test": "npm run eslint && npm run coverage",
     "e2e-single-node": "./test/e2e/single-node.sh",
-    "e2e-multi-node": "sudo ./test/e2e/multi-node.sh"
+    "e2e-single-node-couch3": "./test/e2e/single-node-3.sh",
+    "e2e-multi-node": "sudo ./test/e2e/multi-node.sh",
+    "e2e-multi-node-couch3": "sudo ./test/e2e/multi-node-3.sh"
```

Comment:
Is sudo needed here? It's causing issues on my machine as it makes `~/.docker/buildx/current` be owned by root

Reply:
It's required locally. in CI everything is sudo by default.